src/StackExchange.Redis/PhysicalBridge.cs

```diff
@@ -576,6 +576,7 @@ internal void OnHeartbeat(bool ifConnectedOnly)
                                 else
                                 {
                                     OnDisconnected(ConnectionFailureType.SocketFailure, tmp, out bool ignore, out State oldState);
+                                    using (tmp) { } // dispose etc
```

Comment:
Well, there's no real "etc" here (if we ignore the `try`/`finally` since the `try` is empty) - `tmp.Dispose();` (or `?.`) would seem more direct and would avoid the comment? But I don't care as such

Reply:
Was mirror above there only - agree on simplifying (did both). Also added some more comments to hopefully help later - take another peek pretty please?